theme.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://schemas.wp.org/wp/6.0/theme.json",
-  "version": 2,
+  	"$schema": "https://schemas.wp.org/wp/6.0/theme.json",
```

Comment:
Would you be able to set your editor config so that it doesn't use a mix of spaces and tabs in this file? It looks like the original was tabs and this PR introduced spaces. It would be nice to pick one and be consistent across all of our PRs.

Reply:
Done, no formatting changes in new PR